waterloop-site/src/pages/Recruitment.tsx

```diff
@@ -25,8 +33,19 @@ const Recruitment: React.FC = () => (
         imgData={imgData}
       ></TextWithImage>
       <div className="break"></div>
-      <h2 className={"center-text"}>Current Postings</h2>
-      <JobPostings />
+      {currentlyRecruiting ? (
+        <>
+          <h2 className={"center-text"}>Current Postings</h2>
+          <JobPostings />
+        </>
+      ) : (
+        <FlexContainer>
+          <h3>Our winter member recruitment is now <b>closed</b> for the Fall term.</h3>
```

Comment:
I think you should remove "winter" right? @Quinn1876 
Should just be "Our member recruitment is now closed for the Fall term."

Reply:
It was mentioned by Jenny in the leads channl that we at least keep that on so people would stop messaging us lol